pkg/timeline/adaptor/repository/dummy.ts

```diff
@@ -124,6 +124,15 @@ export class InMemoryListRepository implements ListRepository {
     return Result.ok(list);
   }
 
+  async fetchListsByMemberAccountID(
+    accountID: AccountID,
+  ): Promise<Result.Result<Error, List[]>> {
+    const lists = [...this.listData].filter((list) =>
+      list[1].getMemberIds().includes(accountID),
+    );
+    return Result.ok(lists.map((list) => list[1]));
```

Comment:
```suggestion
    const lists = [...this.listData.values()].filter((list) =>
      list.getMemberIds().includes(accountID),
    );
    return Result.ok(lists);
```
値しか使わないのであれば, `values()` メソッドのイテレータを配列にスプレッドしたほうがよさそうです.